docs/maintenance/ISSUES.md

```diff
@@ -101,3 +101,12 @@ TODO: This will be filled out... soon!
 ### 🚀 New Rules
 
 TODO: This will be filled out... soon!
+
+## Pruning Old Issues
+
+Every so often, we like to [search for open issues `awaiting response`](https://github.com/typescript-eslint/typescript-eslint/issues?q=is%3Aopen+is%3Aissue+label%3A%22awaiting+response%22) to find ones that might have been forgotten.
+Our flow for issues that have been waiting for >=1 month is:
+
+1. Ping the author with a message like the _Checking In_ template
+2. Wait 2 weeks
```

Comment:
maybe add the `stale` tag as well so that we can easily search for prs/issues that are awaiting the 2 week close?

Reply:
I like it!